features/edit_category.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Create Catogory
```

Comment:
Same question, this is an edit_category file, but it seems like this stuff is in the manage_categories feature file?